assets/kube-apiserver/kube-apiserver-configmap.yaml

```diff
@@ -6,7 +6,7 @@ data:
   aggregator-client-ca.crt: |-
 {{ include_pki "root-ca.crt" 4 }}
   kubelet-client-ca.crt: |-
-{{ include_pki "root-ca.crt" 4 }}
+{{ include_pki "combined-ca.crt" 4 }}
```

Comment:
the kubelet is serving with a cert signed by the `cluster-signer` CA, the `combined-ca` is needed for the client to properly validate the kubelet cert.